templates/project/__PROJECT_NAME__/__PROJECT_NAME__-Info.plist

```diff
@@ -45,5 +45,25 @@
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>UTExportedTypeDeclarations</key>
```

Comment:
I'm curious if this also works if you use `UTImportedTypeDeclarations` as the key

I don't think we want to declare all Cordova apps as owners of the WebAssembly file type by default, so `UTExportedTypeDeclarations` seems like it might be the wrong key to use. Even with that though, I think injecting this when needed using `config-file` in config.xml for your app is probably a better choice that unconditionally including this declaration in the template for all Cordova apps.

Reply:
I like your idea of using `UTImportedTypeDeclarations` key instead! I tested it and I can confirm that it works as well.

I understand there are differing views on whether loading WebAssembly code should be supported by default or only when needed.

I would argue for making this a default because I assume quite a few developers will be unaware of the problem until it suddenly occurs in their Testflight or release build. The cause is not easy to find, because the problem does not seem to occur in debug builds, which makes debugging difficult. This could be confusing and will cost time. At the same time, I don't see any downside how the `UTImportedTypeDeclarations` entry could cause problems.

Nevertheless, at least the documentation should point out the WASM loading error in release builds and explain how to fix it when needed using `config-file` in `config.xml`.

Reply:
Maybe it's optimistic, but it would be great if we could have a script to detect if the code is using WASM or not, then warn users to enable usage of WASM in config.xml. But for now this could be enough to somehow easily enable WASM support in config.xml. Putting all those config lines in config.xml is not a minimal way to enable WASM. Maybe a boolean preference like `EnableWASM` makes sense.

`<preference name="EnableWASM" value="true" />`